model/src/main/java/com/epam/esm/entities/GiftCertificate.java

```diff
@@ -0,0 +1,114 @@
+package com.epam.esm.entities;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class GiftCertificate extends Entity {
+
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private int duration;
```

Comment:
What's the purpose of field duration? You should use correct combination of field name and type for anyone to understand the meaning of it at glance. In case it is not clear even after that you have to provide a comment with explanation.